src/_posts/aframe-v0.3.0.md

```diff
@@ -0,0 +1,130 @@
+
+Other remarkable changes include large performance improvements, better support
+for webpage-embedded scenes, and support for multiple instances of components
+of the same type.
+
+## What Have People Built?
+
+A-Frame has grown popular as the WebVR framework of choice. In roughly over
+half a year, the [GitHub repository][github] has over **2500 stargazers and 50
+contributors**. There are over **1100 members** on the [A-Frame Slack
+channel][slack], which has hundreds of more members than the [WebVR Slack
+channel][webvr-slack].
+
+We started producing weekly roundups with the *[A Week of A-Frame][blog]*
+series to showcase community content, components, and contributions.
+
+In the last four months, we have featured over **130 scenes**:
+
+<div class="post__projects">
+  <a href="https://sandbox.donmccurdy.com/vr/island/"
+     title="MagicaVoxel Island"
+     style="background-image: url(/images/awoa/island.gif)"></a>
+  <a href="https://ngokevin.github.io/aframe-audio-visualizer-components/spectrum/"
+     title="Audio Visualizer"
+     style="background-image: url(/images/awoa/audio-visualizer-components-update.gif)"></a>
+  <a href="http://tbaloo.com/tumbvr/madeinhexels"
+     title="TumbVR"
+     style="background-image: url(/images/awoa/tumbvr.jpg)"></a>
+  <a href="https://musicpua.firebaseapp.com/"
+     title="Virtual Symphony"
+     style="background-image: url(/images/awoa/virtual-symphony-2.gif)"></a>
+  <a href="http://www.skyislandsvr.com/pages/SadIsland.html"
+     title="Sad Island"
+     style="background-image: url(/images/awoa/sad-island.jpg)"></a>
+</div>
+
+...and nearly **40 community components**:
+
+- **Component Packs:** [aframe-extras][extras] by [Don McCurdy][don] enables
+  additional **controls**, **physics**, and loaders. [k-frame][k-frame]
+  enables **audio visualizations**, **multiuser**, **templating**, layout, text,
+  and improved animations.
+- **Geometries:** [Procedural geometries][lsystem], [Bitmap font text][bmfont],
+  [terrains][terrain], [oceans][extras], and [grids][grid].
+- **Materials:** [GIF][gif], [HTML][html], [cubemaps][cubemap],
+  [stereo cubemaps][stereocube], [video controls][vidcontrols].
+- **Miscellaneous:** [Leap Motion controls][leap], [Vive controls][webvrcontroller],
+  [AltSpaceVR integration][altspace], [glTF][gltf], [particle systems][particle].
+
+## What's Next?
+
+We want to pull in the **larger community** of artists and creators. A-Frame is
+focused on web developers, but we will create easy-to-follow tutorials and
+workflows for tools to use alongside A-Frame, such as with
+[MagicaVoxel][magicavoxel] and [Blender][blender].
+
+We are excited to improve the **component discovery and consumption process**.
+Rather than having to search for component builds on GitHub, downloading them,
+and manually including them, we hope to be able to lazy-load from a curated
+directory of components. You will be able to use third-party components *as if
+they shipped with A-Frame*.
+
```

Comment:
Nit:

> Rather than having to search for component builds on GitHub, downloading them, and manually including them, we hope to be able to lazy-load from a curated directory of components.

Suggestion: "Rather than having to search for, download, and manually include components from GitHub,  we hope to be able to lazy-load from a curated directory of components."